src/Microsoft.Azure.WebJobs.Logging/Internal/LogWriter.cs

```diff
@@ -237,6 +250,9 @@ public Task AddAsync(FunctionInstanceLogItem item, CancellationToken cancellatio
                     }
                 }
 
+                // Just making a note while I'm in this code fixing something else. 
+                // It looks like this will drop data if there is any type of error when communicating with storage. 
+                // Other code paths drop the data after successfully writing to storage
```

Comment:
Can you file an issue so we can track it?

Reply:
Filed https://github.com/Azure/azure-webjobs-sdk/issues/1761